tests/php/Dev/CsvBulkLoaderTest.php

```diff
@@ -214,10 +214,10 @@ public function testLoadWithCustomHeaderAndRelation()
         $this->assertEquals($testPlayer->ContractID, $testContract->ID, 'Creating new has_one relation works');
 
         // Test nested setting of relation properties
-        $contractAmount = DBField::create_field('Currency', $compareRow[5])->RAW();
+        $contractAmount = DBField::create_field('Currency', $compareRow[5])->Nice();
         $this->assertEquals(
-            $testPlayer->Contract()->Amount,
             $contractAmount,
+            $testPlayer->Contract()->dbObject('Amount')->Nice(),
```

Comment:
This [was failing](https://travis-ci.org/kinglozzer/sapphire/jobs/395006181) because the database query was returning `20000.00` while the manually constructed `DBField` instance was returning `20000`. I don’t know why, but that “regression” only shows up when the statement caching is removed. Given that the value is a `DECIMAL(9, 2)` I’d argue that `20000.00` is more correct than `20000` which is currently returned.